packages/features/ee/common/components/v2/LicenseRequired.tsx

```diff
@@ -18,7 +18,7 @@ const LicenseRequired = ({ children, as = "", ...rest }: LicenseRequiredProps) =
   const session = useSession();
   const { t } = useLocale();
   const Component = as || Fragment;
-  const hasValidLicense = session.data ? session.data.hasValidLicense : null;
+  const hasValidLicense = session.data ? session.data.license.valid : null;
```

Comment:
License validity within the session object has been moved to its own object together with the key in order to have it at hand for other usages.